src/neo/SmartContract/InteropService.Encode.cs

```diff
@@ -0,0 +1,41 @@
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.VM;
+using Neo.Wallets;
+using System;
+using System.Linq;
+
+namespace Neo.SmartContract
+{
+    partial class InteropService
+    {
+        public static class Encode
+        {
+            /// <summary>
+            /// Convert public key to corresponding wallet account address
+            /// </summary>
+            public static readonly InteropDescriptor PubKeyToAddress = Register("Neo.Encode.PubKeyToAddress", Encode_PubKeyToAddress, 0_00010000, TriggerType.All, CallFlags.None);
```

Comment:
This should be removed. We don't use addresses in a contract.

Reply:
> We don't use addresses in a contract.

This is why we want to add.  Use address is more convenient for developers/our testers and more readable for users, like voting dapps, nep5.  
I think we can add the following interopable services:

```js
Neo.Wallet.PrivateKeyToPublicKey
Neo.Wallet.PublicKeyToScriptHash
Neo.Wallet.PublicKeyToAddress
Neo.Wallet.AddressToScriptHash
Neo.Wallet.ScriptHashToAddress
```

But if you prefer to remove, we can do it.

Reply:
This file should be removed. We don't need addresses in contract. We should convert it outside the contract.